tests/test_pinky.rs

```diff
@@ -68,5 +58,8 @@ fn test_short_format() {
 
 #[cfg(target_os = "linux")]
 fn expected_result(args: &[&str]) -> String {
-    TestScenario::new(UTIL_NAME).cmd(UTIL_NAME).args(args).run().stdout
+    use std::process::Command;
+
+    let output = Command::new(UTIL_NAME).args(args).output().unwrap();
```

Comment:
what is the reasoning for this revert? .cmd(UTIL_NAME) has the same effect, but it also runs pinky with a cleared environment.